docker/pool/builder/lib/builder/docker.rb

```diff
@@ -4,17 +4,21 @@
 module Builder
   module Docker
     CONTAINER_PORT = 80
+    ::Docker.options[:read_timeout] = 15 * 60 # 15 minutes
 
+    attr_accessor :logger
+    module_function :logger, :logger=
```

Comment:
@ainoya Could you tell me what do you want to do in this line? I understand what you want to do in `attr_accessor :logger` but I could not catch these:

* why `module_function` is needed 
* why `module_function` gets symbol which is not related to method name